### PR TITLE
Goerli deploy script

### PR DESCRIPTION
**Pull request by 0x0aa0**

### CODE REVIEW

File: contracts/script/deploy/goerliv2/GoerliV2_Deploy.s.sol [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+            IRegistryCoordinator.OperatorSetParam[] memory operatorSetParams, 
+            address churner, 
+            address ejector
+        ) = _parseRegistryCoordinatorParams(config_data);
+
+        //upgrade the registry coordinator proxy to implementation
+        eigenDAProxyAdmin.upgradeAndCall(
+            TransparentUpgradeableProxy(payable(address(registryCoordinator))),
+            address(registryCoordinatorImplementation),
+            abi.encodeWithSelector(
```
Comment (gpsanant) [inline contracts/script/deploy/goerliv2/GoerliV2_Deploy.s.sol:161]: wont these initializers fail?

Comment (0x0aa0) [reply contracts/script/deploy/goerliv2/GoerliV2_Deploy.s.sol:161]: why would they fail?

File: contracts/test/unit/EigenDABlobUtils.t.sol [hunk 2]
```diff
@@ -59,6 +59,7 @@ contract EigenDABlobUtilsUnit is BLSMockAVSDeployer {
                     abi.encodeWithSelector(
                         EigenDAServiceManager.initialize.selector,
                         pauserRegistry,
+                        0,
```
Comment (ian-shim) [inline contracts/test/unit/EigenDABlobUtils.t.sol:62]: just to confirm, it means unpaused right?

**Review by ian-shim — APPROVED** [review]

File: contracts/test/unit/EigenDABlobUtils.t.sol (hunk 2 above)
Comment (0x0aa0) [reply contracts/test/unit/EigenDABlobUtils.t.sol:62]: yes unpaused

**Pull request merged**